examples/js/nodes/materials/StandardNode.js

```diff
@@ -135,7 +137,7 @@ THREE.StandardNode.prototype.build = function( builder ) {
 		var normal = this.normal ? this.normal.buildCode( builder, 'v3' ) : undefined;
 		var normalScale = this.normalScale && this.normal ? this.normalScale.buildCode( builder, 'fv1' ) : undefined;
 
-		var environment = this.environment ? this.environment.buildCode( builder.setCache( 'env' ), 'c' ) : undefined;
+		var environment = this.environment ? this.environment.buildCode( builder, 'c', 'env', requires ) : undefined;
 		var reflectivity = this.reflectivity && this.environment ? this.reflectivity.buildCode( builder, 'fv1' ) : undefined;
```

Comment:
`MeshStandardMaterial` no longer has a `reflectivity` property. Is the goal to keep `MeshStandard` and `StandardNode` in sync?